docs/core/tutorials/consuming-library-with-visual-studio.md

```diff
@@ -6,24 +6,23 @@ ms.author: wiwagn
 ms.date: 06/05/2018
 ms.custom: "vs-dotnet, seodec18"
 ---
-
 # Consume a .NET Standard library in Visual Studio 2017
 
-Once you've created a .NET Standard class library by following the steps in [Building a C# class library with .NET Core in Visual Studio 2017](./library-with-visual-studio.md) or [Building a Visual Basic class library with .NET Core in Visual Studio 2017](vb-library-with-visual-studio.md), tested it in [Testing a class library with .NET Core in Visual Studio 2017](testing-library-with-visual-studio.md), and built a Release version of the library, the next step is to make it available to callers. You can do this in two ways:
+Once you've created a .NET Standard class library by following the steps in [Building a C# class library with .NET Standard in Visual Studio](./library-with-visual-studio.md) or [Building a Visual Basic class library with .NET Core in Visual Studio 2017](vb-library-with-visual-studio.md), tested it in [Testing a class library with .NET Core in Visual Studio 2017](testing-library-with-visual-studio.md), and built a Release version of the library, the next step is to make it available to callers. You can do this in two ways:
```

Comment:
It looks like a couple "2017" keys stayed here. Do you want to fix them now, or wait until we update that content?

```suggestion
Once you've created a .NET Standard class library by following the steps in [Building a C# class library with .NET Standard in Visual Studio](./library-with-visual-studio.md) or [Building a Visual Basic class library with .NET Core in Visual Studio](vb-library-with-visual-studio.md), tested it in [Testing a class library with .NET Core in Visual Studio](testing-library-with-visual-studio.md), and built a Release version of the library, the next step is to make it available to callers. You can do this in two ways:
```